gatling-maven-plugin/src/main/java/io/gatling/mojo/GatlingMojo.java

```diff
@@ -112,6 +112,14 @@
   private boolean failOnError;
 
   /**
+   * Continue execution of simulations despite assertion failure. If you have
+   * some stack of simulations and you want to get results from all simulations
+   * despite some assertion failures in previous one.
+   */
+  @Parameter(property = "gatling.continueAfterAssertionFailure", defaultValue = "false")
+  private boolean continueAfterAssertionFailure;
```

Comment:
My original comment was lost. I intended `continueOnAssertionFailure` ("On" instead of "After").